clang/docs/ReleaseNotes.rst

```diff
@@ -622,6 +622,8 @@ Improvements to Clang's diagnostics
 - Clang no longer emits a "declared here" note for a builtin function that has no declaration in source.
   Fixes #GH93369.
 
+- Clang now has an improved error message when trying to capture a variable for lambda function expressions.
```

Comment:
This should be more descriptive, i.e. I’d mention that this is specifically about the case of trying to capture a class member.